### PR TITLE
germinal: init at 26

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by grahamc** [comment]
Hey there, can you take a look-see at https://github.com/NixOS/nixpkgs/issues/108568#issuecomment-755364223?

File: pkgs/applications/terminal-emulators/germinal/default.nix [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+stdenv.mkDerivation rec {
+  pname = "germinal";
+  version = "26";
+
+  src = fetchFromGitHub {
+    owner = "Keruspe";
+    repo = "Germinal";
+    rev = "v${version}";
+    sha256 = "sha256-HUi+skF4bJj5CY2cNTOC4tl7jhvpXYKqBx2rqKzjlo0=";
```
Comment (andir) [inline pkgs/applications/terminal-emulators/germinal/default.nix:21]: IIRC we are trying to not use the `sha-…` type of hashes in nixpkgs. At least not when assigning to the `sha256` attribute.

Comment (lukegb) [reply pkgs/applications/terminal-emulators/germinal/default.nix:21]: Is this true? I've mostly just given up and started putting any type of hash (sha256: prefixed base32, non-prefixed base32, hex, these SRI hashes...) in any of sha256|hash depending on what time of day it is and how hungry I am...

Comment (AndersonTorres) [reply pkgs/applications/terminal-emulators/germinal/default.nix:21]: @andir I am using the nix command, and it throws out hashes in SRI format now.

**Comment by AndersonTorres** [comment]
Now I think it is done. I need to learn more about wrappers!

**Comment by AndersonTorres** [comment]
Result of `nixpkgs-review pr 108589` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>germinal</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108589` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>germinal</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 108589` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>germinal</li>
  </ul>
</details>

**Pull request merged**